website/templates/nav.mako

```diff
@@ -48,12 +48,12 @@
             <li class="navbar-donate-button"><a data-bind="click: trackClick.bind($data, 'Donate')" href="https://cos.io/donate">Donate</a></li>
             % if user_name and display_name:
             <li class="dropdown">
-            <button class="dropdown-toggle nav-user-dropdown btn-link" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="Toggle auth dropdown">
+            <a class="dropdown-toggle nav-user-dropdown btn-link" data-toggle="dropdown" role="button" aria-expanded="false" aria-label="Toggle auth dropdown">
```

Comment:
This causes the nav bar to be too high because bootstrap adds 15px top and bottom padding around `.navbar-nav > li > a`. One solution will be to adjust the line height in osf-style to 27px (the height of the gravatar image) and reduce top and bottom padding accordingly.

Reply:
this is fixed in osf-style 1.5.1

Reply:
You can actually just drop the `nav-user-dropdown` class and let osf-style set the padding. That's what we're doing in Ember.